client/go/cmd/config.go

```diff
@@ -55,7 +55,7 @@ var setConfigCmd = &cobra.Command{
 }
 
 var getConfigCmd = &cobra.Command{
-	Use:     "get option",
+	Use:     "get <option>",
```

Comment:
Should be `<[option]>` as omitting it prints all options.

Reply:
done